widgets/contact/contact.php

```diff
@@ -832,6 +832,11 @@ function contact_form_action( $instance, $storage_hash ){
 			}
 		}
 
+		// Add in the default subject if no subject field is defined in the form at all
+		if ( !isset( $email_fields['subject'] ) && !empty($instance['settings']['default_subject']) ) {
```

Comment:
Would it be possible to space the second brackets to conform with [WordPress PHP coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)?